chia/types/mempool_item.py

```diff
@@ -36,6 +35,15 @@ def fee_per_cost(self) -> float:
     def name(self) -> bytes32:
         return self.spend_bundle_name
 
+    @property
+    def cost(self) -> uint64:
+        assert self.npc_result.conds is not None
+        return uint64(self.npc_result.conds.cost)
+
+    @property
+    def additions(self) -> List[Coin]:
```

Comment:
This alters some RPCs as they won't include these in their responses anymore.

Reply:
you have a PR to address that, do you think we could get that landed?

Reply:
I think so: https://github.com/Chia-Network/chia-blockchain/pull/14476